comms.py

```diff
@@ -0,0 +1,62 @@
+from mcu import *
+
+class Communications:
+    
+    PWM_MIN = 1000
+    PWM_MID = 1500
+    PWM_MAX = 2000
+    
+    MOTOR_CLAW = 0 
+    MOTOR_FRONT = 1
+    MOTOR_RIGHT = 2
+    MOTOR_LEFT = 3 
+    MOTOR_BACK = 4
+    
+    INITIAL_PERCENT = 100
+    MULTIPLIER_PERCENT = 2
+    
+    def __init__(self, mcuVAR: MCUInterface):
+        self.mcuVAR = mcuVAR
+    
+    def read_send(key_pressed):
+        
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_CLAW,1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_FRONT, 1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_LEFT, 1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_RIGHT, 1000)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_BACK, 1000)
+
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_CLAW, PWM_MID)
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_FRONT, PWM_MID)
+        self.mcuVAR.cmd_setMotorMicroseconds(MOTOR_LEFT, PWM_MID)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_RIGHT, PWM_MID)
+        self.mcuVAR.cmd_setMotorCalibration(MOTOR_BACK, PWM_MID)
+        
+        dict_motors = {
+            
+            "e": MOTOR_CLAW,
+            "d": MOTOR_RIGHT,
+            "w": MOTOR_FRONT,
+            "a": MOTOR_LEFT,
+            "s": MOTOR_BACKWARD
```

Comment:
It doesn't work this way. To go forward, you would ideally make both the left and the right motors move together simultaneously. To go up and down, you would want to make both the forward and backward motors move together simultaneoulsy.

Reply:
What combination of motors move the bot right and left?

Reply:
To turn left and right, there are two ways you can do it, one is to just move the right or the left motors forward. The other is to move the right and left motors in the opposite directions.

Reply:
So this is just a pivot, not actual movement in that direction

Reply:
It's a turn, yes.